### DISCUSSION TITLE
Tamildhool & Tamilhdbox - Tamil TV Shows & TV Serials

### DISCUSSION THREAD

**Discussion by SukunaVirus** [mhdzumair/tamilblasters_stremio_addon#26] [General]
Can you also do a stremio addon for tamildhool and tamilhdbox, its because my parents wants to watch TV shows pls help us! :)

**Comment by mhdzumair** [comment]
What is the link for tamilhdbox?  When i check https://tamilhdbox.com/ 
![image](https://github.com/mhdzumair/tamilblasters_stremio_addon/assets/44891939/15a731bb-51fa-42a2-8618-5f0ade51aa3e)

**Comment by SukunaVirus** [comment]
I dont know bro it's not working now but here's the tamildhool link:
https://www.tamildhool.net/

**Comment by SukunaVirus** [comment]
If u can't do with tamildhool i found 2 more working sites:

;)

ThiraiThee: https://www.thiraithee.net/

Sky Tamil: https://www.skytamil.net/

Thanks bro for trying to help :)
